dnsext-full-resolver/DNS/Cache/Iterative.hs

```diff
@@ -570,8 +571,12 @@ resolveJustDC dc n typ
     where
       mdc = maxNotSublevelDelegation
 
--- ドメインに対する NS 委任情報
-type Delegation = (Domain, NE DEntry)
+{- delegation information for domain -}
+type Delegation =
```

Comment:
Why don't you use `data`?

Reply:
https://github.com/kazu-yamamoto/dnsext/pull/78/commits/b07c5347a73d391db5b234eb517d46dcdf5f64d2